gammapy/estimators/utils.py

```diff
@@ -461,3 +462,127 @@ def compute_lightcurve_doublingtime(lightcurve, flux_quantity="flux"):
     )
 
     return table
+
+
+def get_edges_fixed_bins(fluxpoint, group_size, axis_name="energy"):
+    """Rebin the flux point to combine value adjacent bins
+
+    Parameters
+    ----------
+    fluxpoint : `FluxPoints`
+        The flux point to rebin
+    group_size : int
+        Number of bins to combine
+
+    Returns
+    -------
+    edges_min : `Quantity` or `Time`
+    edges_max : `Quantity` or `Time`
+        Low and High edges of the new axis
+    """
+
+    ax = fluxpoint.geom.axes[axis_name]
+    nbin = ax.nbin
+    if not isinstance(group_size, int):
+        raise ValueError("Only integer number of bins can be combined")
+    idx = np.arange(0, nbin, group_size)
+    if idx[-1] < nbin:
+        idx = np.append(idx, nbin)
+    edges_min = ax.edges_min[idx[:-1]]
+    edges_max = ax.edges_max[idx[1:] - 1]
+    return edges_min, edges_max
+
+
+def get_edges_min_sqrt_ts(fluxpoint, sqrt_ts_threshold, axis_name="energy"):
+    """Rebin the flux point to combine adjacent bins
+       till a minimum significance is obtained
+
+    Parameters
+    ----------
+    fluxpoint : `FluxPoints`
+            The flux point to rebin
+    sqrt_ts_threshold : float
+        The minimum significance desired
```

Comment:
Here I am bit annoyed.

Summing `ts` is indeed the correct approach to combine the max likelihoods. But one should keep in mind that the number of do.f. increases when doing so. Using the square root of the summed `ts` is therefore not a significance in the usual sense. 

So you could either define a minimum threshold in `ts` and not bother with the number of `dof` avoiding the reference to significance.
Conversely you can convert the summed `ts` to a p-value accounting for the number of `dof` and transform it back to a significance. One possible issue here is the fact that p-value calculation will quickly yield 0 for large `ts`.

Reply:
Sorry about missing the d.o.f.

Completely agree about the  p-value calculation for large TS. But If we define a minimum threshold in TS, we should also tell the user the number of bins combined. I can't find a good place for that.

Reply:
I guess we could add a column with dof? This is interesting in other situations as well. However this should be an independent PR.

Reply:
Added